### ISSUE TITLE
Missing reflex/simd.h file 

### ISSUE THREAD

**Issue by m-hm** [Genivia/RE-flex#112]
/usr/local/include/reflex/absmatcher.h:53:10: fatal error: reflex/simd.h: No such file or directory
 #include <reflex/simd.h>

```
$ reflex -V                                                          ⏎
reflex 3.0.9 x86_64-pc-linux-gnu
```

**Comment by genivia-inc** [comment]
The `simd.h` file is part of the repo. This could mean that RE/flex wasn't installed with `make install` to copy the files to the usual locations. If you use a local installation, make sure the correct paths are used with c++ option `-I`.

**Comment by genivia-inc** [comment]
`simd.h` appears to be missing from the installation step, so I'll fix this,

**Comment by genivia-inc** [comment]
Should be fixed now in v3.0.10. Thanks for reporting this!

**Comment by m-hm** [comment]
> Should be fixed now in v3.0.10. Thanks for reporting this!

Thank you
